### ISSUE TITLE
why no video quality & video format selection (kinda like audio) for download?

### ISSUE THREAD

**Issue by demonluo** [obscurename/youtube-dl-gui#3]
video quality should have drop down menu consist of best/1080p/720p/480p/360p/240p etc that favor mp4, if no mp4 or that particular video resolution not found then just aim for the best...
eg,
best:
-f "bestvideo[ext=mp4]+bestaudio[ext=m4a]/best[ext=mp4]/best"
1080p:
-f "bestvideo[height<=1080][ext=mp4]+bestaudio[ext=m4a]/best[height<=1080][ext=mp4]/best"
720p:
-f "bestvideo[height<=720][ext=mp4]+bestaudio[ext=m4a]/best[height<=720][ext=mp4]/best"
480p:
-f "bestvideo[height<=480][ext=mp4]+bestaudio[ext=m4a]/best[height<=480][ext=mp4]/best"
360p:
-f "bestvideo[height<=360][ext=mp4]+bestaudio[ext=m4a]/best[height<=360][ext=mp4]/best"
240p:
-f "bestvideo[height<=240][ext=mp4]+bestaudio[ext=m4a]/best[height<=240][ext=mp4]/best"

btw, the above mention favor mp4 coz the default currently is -f "bestvideo+bestaudio" but some youtube video will end up getting webm if the bestvideo/bestaudio is webm & if bestvideo is mp4 but bestaudio is webm will end up getting mkv after merging...

eg of getting webm if using only bestvideo/bestaudio
https://www.youtube.com/watch?v=DelhLppPSxY

eg of getting mkv if using only bestvideo/bestaudio
http://www.youtube.com/watch?v=L8eRzOYhLuw

anyway, i hope next version will at least favor mp4...

**Comment by obscurename** [comment]
I did plan on adding format + quality options and more based on what youtube-dl offers. I have not fully researched youtube-dl's entire argument array and did not want to make the program larger (looks, not size) than it is now as I wanted to have it to have a small form factor compared to other downloaders, condensing it down as small as possible.

Sorry for the time I took, I was working on other things. I've read your other open issues and I do plan on getting around to them (and this), but for now I am working on relaxing as I have spent the large portion of the last week + a half preparing for the hurricane that hit Florida. Thanks for your suggestions.
